qa-tests-backend/src/test/groovy/CSVTest.groovy

```diff
@@ -156,13 +156,10 @@ class CSVTest extends BaseSpecification {
     }
 
     @Tag("BAT")
+    // TODO(ROX-29220): Fix the test for fixable cves in component query
+    @IgnoreIf({ Env.ROX_FLATTEN_CVE_DATA == "true" })
```

Comment:
nit: as it does not improve output

```suggestion
    @IgnoreIf(reason = "ROX-29220", value = { Env.ROX_FLATTEN_CVE_DATA == "true" })
```

Reply:
Probably ignoring this one as I see 0 occurrences of this currently.